requirements.txt

```diff
@@ -18,7 +18,7 @@ lru-dict==1.2.0
 PyJWT==2.8.0
 cryptography==41.0.3
 pyOpenSSL==23.2.0
-orjson==3.9.3
+orjson==3.9.4
```

Comment:
Should requirements_test_all.txt have the same change?

Reply:
It's listed implicitly
https://github.com/home-assistant/core/blob/2841cbbed272114c97afa2d0b36f30dca6b564d8/requirements_all.txt#L1-L2

Reply:
Thanks, TIL!